library/src/main/java/org/xmtp/android/library/Client.kt

```diff
@@ -334,10 +337,12 @@ class Client() {
 
         if (v3Client?.textToSign() == null) {
             v3Client?.registerIdentity(null)
-        } else {
+        } else if (account != null) {
             v3Client.textToSign()?.let {
                 v3Client.registerIdentity(account.sign(it))
             }
+        } else {
+            Log.i(TAG, "No signer passed but signer was required.")
```

Comment:
Just soft error for now instead of blowing up if they try to create a bundle without a signer and don't already have v3 keys.